test/setup.js

```diff
@@ -1,4 +1,6 @@
 // const gapiService = require('../app/services/gapi-service')
+process.env.GRANT_PERCENTAGE = 40
```

Comment:
Solar PV related?

Reply:
This was required to run tests locally with `npm run test` rather than via docker.